README.md

```diff
@@ -5,17 +5,21 @@ A list of all source map users.
 ## Generators (tools that create .map files)
 
 + [gen-mapping](https://github.com/jridgewell/gen-mapping)
-  + [Babel](https://github.com/babel/babel)
- 
 
 ## Consumers (tools that read in .map files)
 
-+ [Webkit](https://github.com/WebKit/WebKit/)
-+ [Mozilla Sourcemap](https://github.com/mozilla/source-map)
 + [Chrome Devtools](https://github.com/ChromeDevTools/devtools-frontend)
 + [trace-mapping](https://github.com/jridgewell/trace-mapping)
++ [Webkit](https://github.com/WebKit/WebKit/)
+
+## Transformers (tools or libraries that can both consume and produce .map files)
+
++ [Babel](https://github.com/babel/babel) ([`inputSourceMap`](https://babeljs.io/docs/options#inputsourcemap) to consume)
++ [Closure Compiler](https://github.com/google/closure-compiler) (`--apply_input_source_maps --parse_inline_source_maps` to consume)
++ [Mozilla Sourcemap](https://github.com/mozilla/source-map)
```

Comment:
I wonder if it makes sense to separate the low level libraries from the higher level tools? But that might require some kind of edges between the two (e.g. "Parcel" --> "`@parcel/source-map`").

Reply:
I do like this, I think more separation is good!